search/utils.py

```diff
@@ -385,7 +427,7 @@ def _create_mapping(conn):
         conn.indices.delete_mapping(index=INDEX_NAME, doc_type=DOC_TYPE)
 
     mapping = Mapping(DOC_TYPE)
-
+    mapping.field("id", "integer", index="not_analyzed")
```

Comment:
Nice fix on indexing the ID, by the way. I'm 99% sure `index="not_analyzed"` is a no-op on an integer type, but it probably doesn't hurt anything to keep it there.